packages/vue/src/components/list/SingleDropdownList.jsx

```diff
@@ -167,13 +167,20 @@ const SingleDropdownList = {
 		const renderItemCalc = this.$scopedSlots.renderItem || renderItem;
 		const renderErrorCalc = this.$scopedSlots.renderError || renderError;
 		const renderLabelCalc = this.$scopedSlots.renderLabel || renderLabel;
+		const renderNoResults = this.$scopedSlots.renderNoResults || this.$props.renderNoResults;
 
 		if (renderErrorCalc && this.error) {
 			return isFunction(renderErrorCalc) ? renderErrorCalc(this.error) : renderErrorCalc;
 		}
 
 		if (!this.hasCustomRenderer && this.$data.modifiedOptions.length === 0) {
-			return null;
+			if(renderNoResults && isFunction(renderNoResults)) {
```

Comment:
@anik-ghosh-au7 Optimize the condition. Why are you using `div` here?

Reply:
@bietkul the `<div>` is needed without it the content is not getting rendered, in the video I have shown the issue